pl_examples/test_examples.py

```diff
@@ -25,6 +25,29 @@ def test_gpu_template(cli_args):
         run_cli()
 
 
+@pytest.mark.parametrize('cli_args', ['--epochs 1'])
+@pytest.mark.skipif(not torch.cuda.is_available(), reason="test requires GPU machine")
+def test_imagenet(cli_args):
+    """Test running CLI for an example with default params."""
+    from pl_examples.domain_templates.imagenet import run_cli
+
+    import tempfile
+    import skimage
+
+    with tempfile.TemporaryDirectory as tmpdir:
+        os.makedirs(os.path.join(tmpdir.name, 'train', 'class_0'))
+        os.makedirs(os.path.join(tmpdir.name, 'val', 'class_0'))
+        skimage.io.imsave(os.path.join(tmpdir.name, 'train', 'class_0', 'cameraman.jpg',
```

Comment:
`skimage` is not in dependencies and not needed to ass just for one image, check it `torchvision` has a sample image and if not just make 5 images of random noise for two classes

Reply:
A stupid question: how to run the tests locally.

Reply:
simple `python -m pytest pl_examples`

Reply:
Can you verify me that this test file will be run by the CI. Because my previous commit had a grammar error, but the test was fine.

Reply:
you can see if the test is presented in here - http://35.192.60.23/PyTorchLightning/pytorch-lightning/5355/1/2

Reply:
Thanks. It's there. Please review, I have updated image source